lib/models/answer.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:equatable/equatable.dart';
+
+class Answer extends Equatable {
+  final String id;
+  final String text;
+  final int displayOrder;
+  final String displayType;
```

Comment:
Why don't we use the `DisplayType` here? 🤔

Reply:
@markgravity Do you mean using an enum? I changed to use an enum in the next PR https://github.com/luongvo/flutter-survey/pull/65/files#diff-c149f41c10686502d0127f712053c31429cb39941138d01e2cd62640c2307970R10 👋